[class.mi] Define ambiguous/unambiguous base class (#7306)
cplusplus/draft: source/classes.tex

\end{importgraphic}
\end{note}

\pnum
\indextext{base class!ambiguous}%
\indextext{base class!unambiguous}%

> When class \tcode{B} is a possibly indirect base class of class \tcode{D},
> if there is only one possibly indirect base class subobject of type \tcode{B}
> of a \tcode{D} object, \tcode{B} is an \defnadj{unambiguous}{base class} of \tcode{D},
> otherwise, \tcode{B} is an \defnadj{ambiguous}{base class} of \tcode{D}.

Reviewer: In order of increasing nitpicky-ness:
* The "otherwise" could be read as "when `D` doesn't have a base class of type `B`, or when `D` has multiple `B` bases, `B` is an ambiguous base".
* "base class of class `D`" is telling me twice that `D` is a class - It's just too classy ;) - I think "base class of type `D`" reads a bit better.
* "possibly indirect base class" can be read as either "(possibly indirect) (base class)" or "(possibly indirect base) class". Only one has meaning in C++ making it obviously the intended meaning, but I'd prefer to use "(possibly indirect)" or "possibly-indirect" so the reader doesn't need to analyze the sentence.
How about: 
```suggestion
If an object of class \tcode{D} has a single base class subobject of type \tcode{B},
\tcode{B} is an \defnadj{unambiguous}{base class} of \tcode{D}.
If an object of class \tcode{D} has more than one base class subobject of type \tcode{B},
\tcode{B} is an \defnadj{ambiguous}{base class} of \tcode{D}.
```
